### PR TITLE
Mkdocs - fix broken code sample

### PR DESCRIPTION
**Pull request by cloneko**
https://kintone.github.io/kintone-ui-component/ has brocken code sample.
Please fix it.

### CODE REVIEW

File: docs/Reference/RadioButton.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ export default class Plugin extends React.Component {
 ```
 **Javascript**
 ```
-var radioBtn = new kintoneUIComponent.RadioButton({
+var radioBtn = new kintoneUIComponentComponent.RadioButton({
```
Comment (nguyendankha) [inline docs/Reference/RadioButton.md]: @r3-yonashiro 
Please fix this issue, the code is "kintoneUIComponent", not "kintoneUIComponentComponent"

File: docs/Reference/RadioButton.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ export default class Plugin extends React.Component {
...
```
Comment (nguyendankha) [inline docs/Reference/RadioButton.md]: @r3-yonashiro 
Please fix this issue, the code is "kintoneUIComponent", not "kintoneUIComponentComponent"

**Comment by cloneko** [comment]
@nguyendankha 

Thank you for your review.
fix it!

File: docs/Reference/Dropdown.md [hunk 3]
```diff
@@ -851,10 +851,10 @@ var dropdown = new kintoneUIComponent.Dropdown({
     ],
     value: 'Banana'
 });
- 
+
 var body = document.getElementsByTagName("BODY")[0];
-    body.appendChild(dropdown.render();
-    
+    body.appendChild(dropdown.render()));
```
Comment (nguyendankha) [inline docs/Reference/Dropdown.md]: @r3-yonashiro -san
The ')' before ';' is redundant, please remove it for using.

Comment (cloneko) [reply docs/Reference/Dropdown.md]: fixed! Thanks!

**Comment by nguyendankha** [comment]
@r3-yonashiro -san
@cloneko -san

Thank you for your for feedback. We will update the document later. Please wait for our update.

Regards.

**Pull request merged**